src/Form/DataExportType.php

```diff
@@ -87,7 +88,14 @@ public function buildForm(FormBuilderInterface $builder, array $options)
 
             try {
 
-                $content = $exporter->export($start, $end);
+                switch ($data['format']) {
+                    case 'colisactiv_all':
+                        $content = $exporter->export($start, $end, true);
+                        break;
+                    default:
+                        $content = $exporter->export($start, $end);
+                        break;
+                }
```

Comment:
If you move `$includeAllTasks` to the constructor, you don't need to do this anymore, no specific use case 😉